app/s2i/copy_mvn_repo.sh

```diff
@@ -12,6 +12,19 @@ fi
 
 mkdir -p "${target_dir}/m2"
 cd ${local_repo}
+
+if [[ ! -d "${target_dir}/m2.cache" ]]; then
+    mkdir -p "${target_dir}/m2.cache"
+fi
+
+if [[ ! "$(ls -A ${target_dir}/m2.cache)" ]]; then
+     echo "Going to cache locally Syndesis ${version} dependencies. Clean target directory to refresh the cache."
+     find ./io/syndesis | grep -F -- "/${version}/" | grep -v -- "-sources\." | grep -v -- "-tests\." | xargs tar -c | tar -vx -C "${target_dir}/m2.cache"
```

Comment:
Here if you initialize the `m2/.cache` with dependencies from your local maven (`mvn copy:dependencies`) you add the benefit of avoiding a "download the maven world", and will keep the benefits of layering.